logging/src/main/java/se/fortnox/reactivewizard/logging/LoggingFactory.java

```diff
@@ -36,6 +38,13 @@
     @JsonProperty("levels")
     Map<String, String> levels = new HashMap<>();
 
+    @Valid
```

Comment:
"Valid" annotations is used to validate sub-entities, like the items of a list. Does not apply to Boolean.